src/main/java/knight2103/files/Storage.java

```diff
@@ -0,0 +1,135 @@
+package knight2103.files;
+
+import knight2103.tasks.TaskList;
+import knight2103.tasks.Task;
+import knight2103.tasks.Todo;
+import knight2103.tasks.Deadline;
+import knight2103.tasks.Event;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Scanner;
+
+public class Storage {
+    private final File taskFile;
+
+    public Storage(String filePath) {
+        this.taskFile = new File(filePath);
+    }
+
+    /**
+     * Returns ArrayList<Task>. Reads the file that stores all the tasks and
+     * load them into an ArrayList<Task> to be stored in the bot's list of tasks.
+     *
+     * @return The ArrayList of task that can be stored in the bot's list of tasks.
+     */
+    public ArrayList<Task> load() {
```

Comment:
hmm perhaps loadTasks() would be a clearer naming

Reply:
Oh true, thanks for the suggestion!